README.md

```diff
@@ -56,6 +56,8 @@ Within each plugin configuration stanza, the "plugin_specific_configuration" obj
 | es_version                                  | string        |  elasticsearch   |   "7.x" | set to "6.x" or "7.x" depending on Elasticsearch version                   |
 | bulk_count                                  | int           |  elasticsearch   |      10 | the number of text chunks processed at once for ES full-text indexing      |
 | read_size                                   | int           |  elasticsearch   | 4194304 | the size of individual text chunks processed for ES full-text indexing     |
+| tls_certificate_file                        | string        |  elasticsearch   |      "" | the absolute path to a TLS certificate used for secure communication with elasticsearch. if empty, OS-dependent default paths are used for certificates verification. |
+| authorization_basic_credentials             | string        |  elasticsearch   |      "" | the encoded basic authentication credentials for elasticsearch - i.e. base64_encode(url_encode(username) + ":" + url_encode(password)). only used when when non-empty. recommended when using TLS, but not required |
```

Comment:
Should we expose an option for defining the verification mode?

See https://www.boost.org/doc/libs/1_81_0/doc/html/boost_asio/reference/ssl__verify_mode.html.

Reply:
if you think this is a thing we need to think about - let's make an issue to remember.    feels like it needs more research.

Reply:
Well, I think we can ignore this.

If someone needs it, they'll be sure to open an issue or reach out to us through other means.